src/timeMachine/selectors/index.ts

```diff
@@ -52,6 +52,10 @@ export const getActiveTimeMachine = (state: AppState) => {
   return timeMachine
 }
 
+export const getActiveCellID = (state: AppState) => {
+  return get(getActiveTimeMachine(state), 'view.cellID')
```

Comment:
i understand that this is conventional for this file, but we're trying to move away from using lodash functions - native functions make it almost obsolete. so just grab the active time machine and access the view.cellID directly.

Reply:
Got it! Changing now.